nextflow/nextflow_examples.ipynb

```diff
@@ -0,0 +1,1118 @@
+    "<b>Note:</b> This notebook assumes that you have run <a href=\"../workspace_setup.ipynb\">workspace_setup.ipynb</a> in the parent directory to create necessary workspace resources. Please note that you can skip creating a BigQuery dataset as that won't be used in this notebook's examples.\n",
+    "</div>\n",
+    "     Running the `workspace_setup.ipynb` notebook will create two Cloud Storage buckets for your workspace files with workspace reference names: \n",
+    "\n",
+    " - ws_files   \n",
+    " - ws_files_autodelete_after_two_weeks      \n",
+    "    \n",
+    "The code in this notebook will write output files to the \"autodelete\" bucket by default. Any file in this bucket will be automatically deleted two weeks after it is written. This alleviates the need for you to remember to clean up temporary and example files manually. If you want to write outputs to a durable location, simply change the assignment of the `BUCKET_REFERENCE` variable in the cell below and re-run the notebook. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9f48ea42",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Change this to \"ws_files\" to use the durable workspace bucket instead of the autodelete bucket.\n",
+    "BUCKET_REFERENCE = \"ws_files_autodelete_after_two_weeks\"\n",
+    "\n",
+    "!terra resource resolve --name $BUCKET_REFERENCE || echo \"Be sure to run workspace_setup.ipynb before this notebook.\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "df46666b-f7dc-4595-95f8-4a0f153b0213",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Install `nf-core` tool\n",
+    "\n",
+    "In order to interact with `nf-core` pipelines in this notebook,  you'll need to install the [`nf-core` companion tool](https://nf-co.re/tools), a command-line interface.\n",
+    "\n",
+    "1. Run the first cell below to install the tool in your Terra cloud environment. \n",
+    "2. Once installation has completed, navigate to the **Kernel** dropdown in the JupyterLab menu and select **Restart Kernel**. \n",
+    "3. Once the kernel has restarted, run the second cell below to import the tool."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "881e8faa-a865-4fb3-ba69-7f4df579931a",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    import nf_core\n",
+    "    print(\"nf-core is already installed\")\n",
+    "except:\n",
+    "    print(\"Installing nf-core...\")\n",
+    "    !pip install nf-core"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "450ea83f-f6f5-44d4-9649-7fdcca4c7288",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    import nf_core\n",
+    "    print(\"nf-core is already installed\")\n",
+    "except:\n",
+    "    print(\"Please restart the kernel before importing...\")"
```

Comment:
Lets move these import cells to the very top. Right now, if I run the cells in order and then restart my kernel, I'll lose my BUCKET_REFERENCE variable.